packages/frontend/src/app/repository/tests-list/test-details/test-details.component.html

```diff
@@ -50,8 +50,9 @@
     <!-- FLAKINESS -->
     <span *ngIf="test.flaky" class="flaky">FLAKY</span>
     <span *ngIf="!test.flaky">NOT FLAKY</span>
+    <span>.&nbsp;</span>
 
-    <span>.</span>
+    <button mat-raised-button (click)='startDeleteTest()' id='delete-test-button' class='deleteTestButton'>Delete</button>
```

Comment:
nit: I would probably hyphenate a class name `delete-test-button`.

Reply:
Yep. Also, since there will be multiple instances of this button on the dom, i think it can't have an id (since ids are unique).
Oh, also, i'm not sure its super important, but double quotes (`"`) instead of single quotes? (for the templates only)